docs/en_US/CommunitySharings/HpoComparision.md

```diff
@@ -5,17 +5,17 @@ Comparison of Hyperparameter Optimization algorithms on several problems.
 
 Hyperparameter Optimization algorithms are list below:
 
-- [Random Search](../Builtin_Tuner.md#Random)
-- [Grid Search](../Builtin_Tuner.md#Random)
-- [Evolution](../Builtin_Tuner.md#Evolution)
-- [Anneal](../Builtin_Tuner.md#Anneal)
-- [Metis](../Builtin_Tuner.md#MetisTuner)
-- [TPE](../Builtin_Tuner.md#TPE)
-- [SMAC](../Builtin_Tuner.md#SMAC)
-- [HyperBand](../Builtin_Tuner.md#Hyperband)
-- [BOHB](../Builtin_Tuner.md#BOHB)
-
-All algorithms run in NNI local environment。
+- [Random Search](../BuiltinTuner.md)
+- [Grid Search](../BuiltinTuner.md)
+- [Evolution](../BuiltinTuner.md)
+- [Anneal](../BuiltinTuner.md)
+- [Metis](../BuiltinTuner.md)
+- [TPE](../BuiltinTuner.md)
+- [SMAC](../BuiltinTuner.md)
+- [HyperBand](../BuiltinTuner.md)
+- [BOHB](../BuiltinTuner.md)
+
```

Comment:
Why to remove the `#Tuner` in these links?

Reply:
Please to check another PR. Cannot show #Tuner in readdocs.